RecoTracker/CkfPattern/plugins/GroupedCkfTrajectoryBuilder.cc

```diff
@@ -1077,11 +1075,8 @@ TempTrajectory GroupedCkfTrajectoryBuilder::backwardFit(TempTrajectory& candidat
   //cout << "firstTsos "<< firstTsos << endl;
   firstTsos.rescaleError(10.);
   //TrajectoryContainer bwdFitted(fitter.fit(fwdTraj.seed(),fwdTraj.recHits(),firstTsos));
-  Trajectory&& bwdFitted = fitter.fitOne(
-      TrajectorySeed(
-          PTrajectoryStateOnDet(), TrajectorySeed::recHitContainer(), oppositeDirection(candidate.direction())),
-      fwdTraj.recHits(),
-      firstTsos);
+  Trajectory&& bwdFitted =
```

Comment:
why `Trajectory&&` ?

(I understand it was like that in the original code, but I'm curious, so if anybody has any comments about it...)

Reply:
I don't know, to me it seems unnecessary. I didn't change it in my upcoming commit that address the other comments though, because maybe it has a purpose I'm not aware of.